redux-spotlight/src/views/Counter/counterSlice.js

```diff
@@ -17,10 +18,20 @@ export const counterSlice = createSlice({
     decrement: (state) => {
       state.count -= 1;
     },
+    multiply: (state) => {
+      state.count *= 2;
+    },
+    reset: (state) => {
+      state.count = 0;
+    },
+    darkMode: (state) => {
```

Comment:
I might create a separate reducer for darkMode as it is not really related to the count